zproject/default_settings.py

```diff
@@ -548,6 +548,10 @@
 
 FREE_TRIAL_DAYS: Optional[int] = int(get_secret("free_trial_days", "0"))
 
+# Date at which to schedule upgrade for remote server.
+# IN "%Y-%m-%d-%H-%M-%S" format.
+REMOTE_SERVER_UPGRADE_DUE_DATE: Optional[str] = get_secret("remote_server_upgrade_due_date")
+
```

Comment:
Just set `remote_server_upgrade_due_date` in `dev-secrets` and run `./manage.py populate_billings_realms ----only-remote-server` to test.